src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color = getColor();
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (radius * radius * Math.PI);
```

Comment:
The method name 'areaCalculator' should be a noun, not a verb. Consider renaming it to something like 'calculateArea' to better reflect its purpose as a method.